bin/lint-i18n-translations.js

```diff
@@ -0,0 +1,55 @@
+/*
+    Validating i18n translation keys
+    Translation files should match following rules:
+    1. All supported languages should have the same set of keys
+    2. Each key should be on the same line in JSON-file for each language
+*/
+const fs = require('fs')
+const { diffLinesUnified, diffLinesRaw } = require('jest-diff')
+
+const langDir = 'apps/condo/lang'
+
+const translations = []
+
+fs.readdirSync(langDir).map(fileName => {
+    const lang = fileName.match(/(\w+)\.json/)[1]
+    const filePath = `${langDir}/${fileName}`
+    console.log(`Load file ${filePath}`)
+
+    let data
+    try {
+        data = fs.readFileSync(filePath, 'utf8')
+    } catch (e) {
+        console.log(`Error reading file "${fileName}"`)
+        console.error(e)
```

Comment:
process.exit(1)

Reply:
is you can't read the file it means that something gets wrong.

Reply:
Added